Dockerfile

```diff
@@ -1,30 +1,38 @@
 FROM ruby:2.5.3-slim
 
-RUN apt-get update && apt-get install -y build-essential default-libmysqlclient-dev libpq-dev libsqlite3-dev wget apt-transport-https git curl
-RUN curl -sL https://deb.nodesource.com/setup_6.x | bash - && apt-get install nodejs -y
-RUN curl -fsSL https://get.docker.com | bash -
+# Install dependencies
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+    build-essential \
+    default-libmysqlclient-dev \
+    libpq-dev \
+    libsqlite3-dev \
+    wget \
+    apt-transport-https \
+    git \
+    curl \
+    gnupg2 \
+  && curl -sL https://deb.nodesource.com/setup_6.x | bash - \
+  && apt-get install nodejs -y \
+  && rm -rf /var/lib/apt/lists/* \
+  && curl -fsSL https://get.docker.com | bash -
 
 WORKDIR /app
 
 # Mostly static
-COPY config.ru /app/
-COPY Rakefile /app/
+COPY .ruby-version config.ru Rakefile .env.virtualbox /app/
 COPY bin /app/bin
```

Comment:
We no need to prepend '/app' everywhere, as we are in `WORKDIR /app`, just specify target file or folder.
eg: COPY bin bin
COPY .env.bootstrap .env

Reply:
Still some https://github.com/zendesk/samson/blob/5e97920ef5ed1b1529df91a91154da062730cd8f/Dockerfile#L39